wp-admin/js/widgets/media-widgets.js

```diff
@@ -121,8 +121,34 @@ wp.mediaWidgets = ( function( $ ) {
 							embedLinkView.dfd.abort();
 						}
 
+						fetchSuccess = function( response ) {
+							embedLinkView.renderoEmbed({
+								data: {
+									body: response
+								}
+							});
+
+							$( '#embed-url-field' ).removeClass( 'invalid' );
+							embedLinkView.views.parent.views.parent.views.get( '.media-frame-toolbar' )[0].$el.find( '.media-button-select' ).prop( 'disabled', false );
```

Comment:
Looks like this code is duplicated a few times in the PR here, would a helper method to toggle the button make sense?  Also would be really nice if there was an easier way to access these buttons and disable them.